pkgs/tools/networking/mpssh/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchFromGitHub, openssh }:
+
+stdenv.mkDerivation rec {
+  pname = "mpssh";
+  version = "1.3.3";
+
+  src = fetchFromGitHub {
+    sha256 = "0iibzb9qrwf9wsmp2b8j76qw66ql8fw5jx02x82a22fsaznb5dng";
+    rev = "${version}";
+    repo = pname;
+    owner = "ndenev";
+  };
+
+  buildInputs = [ openssh ];
+
+  postPatch = ''
+    substituteInPlace Makefile --replace /usr/local $out
+    substituteInPlace Makefile --replace "-o root" ""
+    '';
+
+  makeFlags = "CC=cc LD=cc BIN=$(out)/bin SSHPATH=${openssh}/bin/ssh SCPPATH=${openssh}/bin/scp";
+
+  meta = {
```

Comment:
It is usually nicer to use `with stdenv.lib` here, so that it does not need to be repeated in `meta`.